app/helpers/application_helper/toolbar/firmware_registries_center.rb

```diff
@@ -0,0 +1,68 @@
+class ApplicationHelper::Toolbar::FirmwareRegistriesCenter < ApplicationHelper::Toolbar::Basic
+  button_group(
+    'firmware_registries_reloading',
+    [
+      button(
+        :firmware_registries_reload,
+        'fa fa-refresh fa-lg',
+        N_('Refresh this page'),
+        N_('Refresh'),
+        :data => {
+          'function'      => 'location.reload()',
+          'function-data' => '{}',
+        }
+      ),
+    ]
+  )
+  button_group(
+    'firmware_registry_config',
+    [
+      select(
+        :firmware_registry_config_choice,
+        'fa fa-cog fa-lg',
+        t = N_('Configuration'),
+        t,
+        :enabled => true,
+        :items   => [
+          api_button(
+            :firmware_registry_sync,
+            'fa fa-refresh fa-lg',
+            t = N_('Refresh Relationships'),
+            t,
+            :klass   => ApplicationHelper::Button::ButtonWithoutRbacCheck,
+            :api     => { :action => 'sync_fw_binaries', :entity => 'firmware_registries' },
```

Comment:
(style) one line hashes in ruby should *not* have the spaces inside the {}, that's reserved for blocks.

```
            :api     => {:action => 'sync_fw_binaries', :entity => 'firmware_registries'},
```

or

```
            :api     => {:action => 'sync_fw_binaries',
                         :entity => 'firmware_registries'},
```